src/app/fix-for-ngfor/fix-for-ngfor.component.html

```diff
@@ -0,0 +1,15 @@
+<wizard> 
+	<wizard-step title="demo" *ngFor="let wizard of wizardSteps"> 
```

Comment:
Can you extend the `wizardSteps` array to contain json objects with two fields `header` and `content`?
The header can then be used for the `stepTitle` and `content` can be used as the content of the step.